acados/ocp_nlp/ocp_nlp_common.c

```diff
@@ -463,15 +471,17 @@ int ocp_nlp_out_calculate_size(ocp_nlp_solver_config *config, ocp_nlp_dims *dims
     int *nx = dims->nx;
     // int *nu = dims->nu;
     int *ni = dims->ni;
+    int *nz = dims->nz;
 
     int size = sizeof(ocp_nlp_out);
 
     size += 3 * (N + 1) * sizeof(struct blasfeo_dvec);  // ux lam
-    size += 1 * N * sizeof(struct blasfeo_dvec);        // pi
+    size += 2 * N * sizeof(struct blasfeo_dvec);        // pi z
```

Comment:
Maybe stupid question, but why isn't the dimension of z also N+1 like ux?

Reply:
what about this question @giaf, @FreyJo?

Reply:
yeah, we only have N algebraic variables (there is no z_N), but we could make it 1 longer, just like u, although there is no u_N.

Reply:
I would say for consistency to also have it N+1, since in principle u[N] is also supported (perhaps already used since Branimir implemented reformulation of soft constraints to controls)

Reply:
done